### PR TITLE
Response

### PR DESCRIPTION
**Pull request by kellye2010**
Diagnostic submission

### CODE REVIEW

**Comment by kellye2010** [comment]
Pull request for diagnostic changes submitted

File: diagnostic.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ Using Atom, open up `diagnostic.md` and write your answers below (where indicate
 1. Is `/Users/blah_blah/Desktop` a relative path or an absolute path? How do you know?
 
  <!-- Answer Starts Here -->
-
+this is an absolute path
```
Comment (Jcornmanhomonoff) [inline diagnostic.md:58]: Correct, but it asks you to explain why: / represents the root of the system

**Comment by Jcornmanhomonoff** [comment]
2. Relative path, planning is referenced with respect to the current working directory, project.
3. Relative, the image file lives inside our current working directory, project; as a result, an absolute path is not needed to access it.


5/7! Awesome job! Keep up the great work! 🦄 👍

File: diagnostic.md (hunk 1 above)
Comment (kellye2010) [reply diagnostic.md:58]: I need to resubmit. I completed my pull request, but forgot that I had not finished ALL of the questions in their entirety! I will re-do.

**Pull request closed without merging**